### PR TITLE
docs: Update index.zh-CN.md

### PR DESCRIPTION
**Pull request by cc20140820**
添加 autoplaySpeed 和 speed 属性

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/carousel/index.zh-CN.md](https://github.com/cc20140820/ant-design/blob/patch-2/components/carousel/index.zh-CN.md)

### CODE REVIEW

File: components/carousel/index.zh-CN.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ subtitle: 走马灯
 | --- | --- | --- | --- | --- |
 | afterChange | 切换面板的回调 | function(current) | 无 |  |  |
 | autoplay | 是否自动切换 | boolean | false |  |  |
+| autoplaySpeed | 轮播间隔时长 | number | 3000 |  |  |
+| speed | 图片切换速度 | number | 500 |  |  |
```
Comment (dengfuping) [inline components/carousel/index.zh-CN.md]: - `index.en-US.md` should be updated also.

**Review by dengfuping — CHANGES_REQUESTED** [review]

File: components/carousel/index.zh-CN.md (hunk 1 above)
Comment (cc20140820) [reply components/carousel/index.zh-CN.md]: ok,it has been commited just now

**Comment by zombieJ** [comment]
透传自 `react-slick` 的 api 暂时先不要在文档里透出，这个组件的作者缺乏维护，透出过多可能会影响之后的维护 or 迁移。

**Comment by cc20140820** [comment]
copy that

**Pull request closed without merging**